src/main/groovy/me/tatarka/RetrolambdaPluginAndroid.groovy

```diff
@@ -123,7 +123,8 @@ public class RetrolambdaPluginAndroid implements Plugin<Project> {
         }
 
         JavaCompile compileUnitTest = (JavaCompile) project.tasks.find { task -> 
-            task.name.startsWith("compile${variant.capitalize()}UnitTestJava") 
+            task.name.startsWith("compile${variant.capitalize()}UnitTestJava") ||
+            task.name.startsWith("compile${variant.capitalize()}UnitTestJavaWithJavac")
```

Comment:
It should, but for my case it doesn't finds unit test variant's task. Dunno why `startsWith` works in unexpected way. :(

For unitTest variant my task's graph looks:
> :app:compileDevDebugAndroidTestSources
> :app:compileDevDebugUnitTestJavaWithJavac
> :app:compileDevDebugUnitTestXAspectJWithAjc
> :app:processDevDebugJavaRes
> :app:processDevDebugUnitTestJavaRes
> :app:compileDevDebugUnitTestSources

And androidTest variant's graph:
> :app:compileDevDebugAndroidTestJavaWithJavac
> :app:compileDevDebugAndroidTestXAspectJWithAjc
> :app:compileRetrolambdaDevDebugAndroidTest
> :app:compileDevDebugAndroidTestNdk UP-TO-DATE
> :app:compileDevDebugAndroidTestSources

My task `app:compileDevDebugUnitTestXAspectJWithAjc` doesn't have strictly corresponding with retrolambda task lifecycle.